rocketpy/motors/solid_motor.py

```diff
@@ -314,6 +315,11 @@ class Function. Thrust units are Newtons.
             "nozzle_to_combustion_chamber".
         reference_pressure : int, float, optional
             Atmospheric pressure in Pa at which the thrust data was recorded.
+        only_radial_burn : boolean, optional
+            If True, inhibits the grain from burning axially, only computing
+            radial burn. Otherwise, if False, allows the grain to also burn
```

Comment:
Redundant

```suggestion
            radial burn. Otherwise, allows the grain to also burn
```

or 

```suggestion
            radial burn. If False, allows the grain to also burn
```